spec/lib/submission_builder/ty2024/states/md/md_return_xml_spec.rb

```diff
@@ -105,11 +142,134 @@
       end
 
       context "502CR" do
-        it "attaches a 502CR" do
-          expect(xml.at("Form502CR")).to be_present
-          expect(instance.pdf_documents).to be_any { |included_documents|
-            included_documents.pdf = PdfFiller::Md502CrPdf
-          }
+        context "L24" do
+          before do
+            allow(instance).to receive(:form_has_non_zero_amounts).with("MD502CR_", anything).and_return false
```

Comment:
i've never seen `anything` as an argument to pass in, that's interesting!

Reply:
i discovered it as part of this!